specification/migrate/resource-manager/Microsoft.Migrate/stable/2018-02-02/migrate.json

```diff
@@ -1670,9 +1709,183 @@
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/acceptLanguage"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Deleted the projects.",
+            "headers": {
+              "x-ms-request-id": {
+                "type": "string",
+                "description": "Service generated Request ID."
+              }
+            }
+          },
+          "204": {
+            "description": "No content. No project with specified name was found."
+          },
+          "400": {
+            "description": "Bad request. (Invalid subscriptionId)."
+          },
+          "401": {
+            "description": "Not authorized (No access to subscription, resource group, or project)"
+          },
+          "500": {
+            "description": "Internal Server Error."
+          },
+          "503": {
+            "description": "Service Unavailable."
+          }
+        },
+        "x-ms-examples": {
+          "SubscriptionNotification_Delete": {
+            "$ref": "./examples/SubscriptionNotification_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Migrate/locations/{locationName}/assessmentOptions": {
+      "get": {
+        "summary": "Get the assessment options.",
+        "description": "Get the available options for the properties of an assessment.",
+        "operationId": "AssessmentOptions_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/locationName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/acceptLanguage"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns assessment options.",
+            "schema": {
+              "$ref": "#/definitions/AssessmentOptionsResultList"
+            }
+          },
+          "401": {
+            "description": "Not authorized (No access to subscription)."
+          },
+          "404": {
+            "description": "Not found (Invalid subscriptionId)."
+          },
+          "500": {
+            "description": "Internal Server Error."
+          },
+          "503": {
+            "description": "Service Unavailable."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "AssessmentOptions_Get": {
+            "$ref": "./examples/AssessmentOptions_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Migrate/projects": {
+      "get": {
+        "summary": "Get all projects.",
+        "description": "Get all the projects in thh subscription.",
+        "operationId": "ProjectsInSubscription_List",
```

Comment:
This operation should be called `Projects_List` and the operation that lists according to resource group should be called something like `Projects_ListByResourceGroup` - would this change be possible or do you already have published SDKs that will break?
Otherwise, this would be changed to `Projects_ListBySubscription`.

Additionally there's a typo in the description: `thh`.

Reply:
Done.